packages/@lightningjs/ui-components/src/components/TextBox/TextBox.js

```diff
@@ -271,6 +273,14 @@ export default class TextBox extends Base {
     }
   }
 
+  get marqueeProps() {
+    return this._marqueeProps ?? {};
+  }
+
+  set marqueeProps(props) {
+    this._marqueeProps = props;
+  }
```

Comment:
since these are in the `properties` array we should use the `_getMarqueeProps` and `_setMarqueeProps` functions so we aren't overriding the accessors created by `withUpdates`